docs/tutorial/part-zero/index.md

```diff
@@ -1,235 +1,235 @@
+title: Imposta il tuo Ambiente di Sviluppo
 typora-copy-images-to: ./
 disableTableOfContents: true
 ---
 
-Before you start building your first Gatsby site, you’ll need to familiarize yourself with some core web technologies and make sure that you have installed all required software tools.
+Prima di iniziare a creare il tuo primo sito Gatsby, avrai bisogno di familiarizzare con alcune tecnologie web di base e assicurarti di aver installato tutti gli strumenti software richiesti.
 
-## Familiarize yourself with the command line
+## Familiarizzare con la linea di comando
 
-The command line is a text-based interface used to run commands on your computer. You’ll also often see it referred to as the terminal. In this tutorial, we’ll use both interchangeably. It’s a lot like using the Finder on a Mac or Explorer on Windows. Finder and Explorer are examples of graphical user interfaces (GUI). The command line is a powerful, text-based way to interact with your computer.
+La linea di comando è una interfaccia testuale che esegue comandi sul tuo computer. Sarà anche fatto riferimento ad essa come terminale. In questo tutorial, useremo entrambi. È un po come usare il Finder sul Mac o Explorer su Windows. Il Finder e Explorer sono esempi di interfaccia grafica (GUI). La linea di comando è un modo potente, basato sul testo, per interagire con il computer.
 
-Take a moment to locate and open up the command line interface (CLI) for your computer. Depending on which operating system you are using, see [**instructions for Mac**](http://www.macworld.co.uk/feature/mac-software/how-use-terminal-on-mac-3608274/), [**instructions for Windows**](https://www.lifewire.com/how-to-open-command-prompt-2618089) or [**instructions for Linux**](https://www.howtogeek.com/140679/beginner-geek-how-to-start-using-the-linux-terminal/).
+Prenditi un attimo per cercare e aprire la linea di comando (CLI) sul tuo computer. A seconda del sistema operativo che stai usando, controlla [**istruzioni per Mac**](http://www.macworld.co.uk/feature/mac-software/how-use-terminal-on-mac-3608274/), [**istruzioni per Windows**](https://www.quora.com/How-do-I-open-terminal-in-windows) o [**istruzioni per Linux**](https://www.howtogeek.com/140679/beginner-geek-how-to-start-using-the-linux-terminal/).
 
-_Note: If you’re new to the command line, "running" a command, means "writing a given set of instructions in your command prompt, and hitting the Enter key". Commands will be shown in a highlighted box, something like `node --version`, but not every highlighted box is a command! If something is a command it will be mentioned as something you have to run/execute._
+_Nota: Se sei nuovo alla, "lanciare" un comando, significa "scrivere un dato set di istruzioni nel prompt dei comandi, e premere il tasto Enter". I comandi saranno evidenziati, una cosa del tipo `node --version`, ma non ogni cosa evidenziata è un comando! Se qualcosa è un comando, esso verrà indicato come qualcosa da lanciare/eseguire._
```

Comment:
Se sei nuovo alla...?

Enter ha una sua traduzione in italiano (Invio)

Rivedrei la formulazione